renderer/brave_content_renderer_client.cc

```diff
@@ -63,6 +65,27 @@
 #endif
 
 namespace {
+
+void EnsureBraveInternalWorldInitialized() {
+  static bool initialized = false;
+  if (initialized) {
+    return;
+  }
+
+  initialized = true;
+
+  // Set an empty CSP so that the main world's CSP is not used in the isolated
+  // world.
+  constexpr char kContentSecurityPolicy[] = "";
```

Comment:
i thought there was a default CSP for isolated worlds? if not we should set it to be something more restrictive than empty.

Reply:
if we need to set it we should probably start with

```
        "default-src 'none'; frame-ancestors 'none'; base-uri 'none';"                                              
        "connect-src 'self';"                                                   
        "script-src 'self';"                      
```

Reply:
@diracdeltas let's make this change in a dedicated PR?
Some features could violate this policy now => some adjustments are needed.

cc @antonok-edm

Reply:
i just remembered we use the same ISOLATED_WORLD_ID_BRAVE_INTERNAL for a bunch of other stuff (playlist, speedreader, ads, cosmetic filtering). is this change going to affect those features as well?

also it's not clear to me how this code interacts with https://github.com/brave/brave-core/blob/ea08eba2f949b38917742c32707ebf30aa859395/components/cosmetic_filters/renderer/cosmetic_filters_js_render_frame_observer.cc#L48 since AFAICT this is using the same world id

Reply:
yep, if we change CSP here it will affect all there features.
That's why I suggest to split out it from the current PR.